azure-pipelines.yml

```diff
@@ -27,10 +27,10 @@ parameters:
   default:
   - name: AMD64
     value: amd64
-    pool: pool-ubuntu-2004-multi-core
+    pool: pool-ubuntu-latest-multi-core
```

Comment:
Can this file be changed to use `.azure-pipelines/templates/variables.yml`?

Reply:
Resloved in the new PR: https://github.com/Azure/azure-cli/pull/30996